rust/crd/src/authentication.rs

```diff
@@ -143,6 +149,24 @@ pub enum AirflowAuthenticationClassResolved {
     },
 }
 
+impl AirflowAuthenticationClassResolved {
+    pub fn tls_ca_cert_mount_path(&self) -> Option<String> {
+        self.tls_client_details().tls_ca_cert_mount_path()
+    }
+
+    pub fn tls_client_details(&self) -> &TlsClientDetails {
+        match self {
+            AirflowAuthenticationClassResolved::Ldap {
+                provider: ldap::AuthenticationProvider { tls, .. },
+            } => tls,
+            AirflowAuthenticationClassResolved::Oidc {
+                provider: oidc::AuthenticationProvider { tls, .. },
+                ..
+            } => tls,
+        }
+    }
+}
+
```

Comment:
Unused when `AirflowClientAuthenticationDetailsResolved::tls_ca_cert_mount_path` is removed.

```suggestion
```